core/domain/stats_jobs_one_off.py

```diff
@@ -334,7 +334,24 @@ def prepare_reduce(cls, exp_id, values):
         Returns:
             tuple(list(dict), list(ExplorationStats), list(str)). 3-tuple where:
             - The first element is the list of dicts of different version-wise
-                ExplorationStats.
+                ExplorationStats. Each dict contains the following key-value
+                pairs:
+                - num_starts_v1: int. Number of learners who started the
+                    exploration.
+                - num_starts_v2: int. As above, but for events with version 2.
+                - num_actual_starts_v1: int. Number of learners who actually
+                    attempted the exploration. These are the learners who have
+                    completed the initial state of the exploration and traversed
+                    to the next state.
+                - num_actual_starts_v2: int. As above, but for events with
+                    version 2.
+                - num_completions_v1: int. Number of learners who completed the
+                    exploration.
+                - num_completions_v2: int. As above, but for events with
+                    version 2.
+                - state_stats_mapping: dict. A dictionary mapping the state
+                    names of  an exploration to the corresponding StateStats
```

Comment:
Nit: remove space before "an".

Also, just to check -- in the state_stats_mapping subdict, are the values actually StateStats domain objects, or are they dicts?

Reply:
Done. The type info of `state_stats_mapping` is dict which is a mapping from state name to StateStats domain objects. Please see [here](https://github.com/oppia/oppia/blob/919b3d824565a0b2628583356d845caa64a6b371/core/domain/stats_domain.py#L75) for reference.

Reply:
Yeah, I've printed out its type info to cross-check. it's `dict`.

Reply:
Then don't you need to correct the description in this docstring, too?